doc/decisions/uuid.md

```diff
@@ -0,0 +1,29 @@
+# Uuid
+
+## Problem
+
+Due to their randomness UUIDs might be sub-optimal for database operations,
+see https://www.cybertec-postgresql.com/en/unexpected-downsides-of-uuid-keys-in-postgresql/
+
+## Constraints
+
+## Assumptions
+
+1. we are actually hit by the problem
+2. we cannot easily change the UUIDs later in production
+
+## Solutions
+
+Use UUIDv7
+
```

Comment:
I recently stumbled upon ULIDs, which aim to solve the above problem.
https://github.com/ulid/spec

Reply:
Great, thanks for mentioning. I added it in 1331d7cd10d7599a194a7cf027754eba4d5eefbb

Do you have some experience with it? Would you favor it over UUIDv7?

Reply:
I think they are basically the same^^ ULIDs are 10 characters shorter. I think going with UUIDv7 should be fine.